rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -65,7 +65,7 @@ struct watchdog_t
 
 			if (old.active && current_time - start_time >= old.timeout)
 			{
-				sys_game.warning("Watchdog timeout! Restarting the game...");
+				sys_game.notice("Watchdog timeout! Restarting the game...");
```

Comment:
This should upped for error.

Reply:
Uh I don't think it is that serious, since a few games sometimes will **intentionally** stop claering the watchdog in order to make themselves rebooted by the watchdog.
Plus, it's not actually an error occurring in the HLE.

Reply:
And according to https://github.com/RPCS3/rpcs3/wiki/Coding-Style (_Use .warning() method for partially implemented functions which still have some unimplemented functionality._), it isn't a partially implemented function either, so I think .notice() would be more suitable for this situation.

Reply:
Restarting a game due to a potential crash is a very serious IMO.

Reply:
Or success level.

Reply:
> Restarting a game due to a potential crash or hang is very serious IMO.

The message in the game:
![image](https://user-images.githubusercontent.com/17809637/197133435-d41fbd9c-9c79-44d2-a37f-c16eb7aab4a6.png)
It means: _*If successful, a watchdog reset will be applied immediately after_

Reply:
Then success log level.

Reply:
Should we consider this an error?

Reply:
These coding style suggestions only count for initial log messages on function calls.
It is still important to raise awareness if something like this happens.
I think error or warning is quite fitting

Reply:
Okay, I would just keep it as-is (warning).

Reply:
`success` is the most noticable log level that is not an error and is fitting for "normal" game operations that occur not many times within a single game session.

Reply:
Interesting. Just changed it to `success`.